src/api/remote/git/repository.rs

```diff
@@ -72,11 +69,10 @@ impl Error {
     }
 }
 
-/// List references that have been updated in the last 30 days.
+/// List references that have been updated
 #[tracing::instrument]
 pub async fn list_references(transport: &Transport) -> Result<Vec<Reference>, Report<Error>> {
```

Comment:
Reminder: Doc comments should be grammatically well formed (the primary exception being that the subject of the sentence is allowed to be implicitly "the thing the comment is attached to").
```suggestion
/// List all references.
#[tracing::instrument]
pub async fn list_references(transport: &Transport) -> Result<Vec<Reference>, Report<Error>> {
```